README.yaml

```diff
@@ -7,6 +7,22 @@
 # Name of this project
 name: terraform-aws-vpc-peering
 
+# Tags of this project
+tags:
+  - aws
+  - terraform
+  - terraform-modules
+  - platform
+  - vpc
+  - ecommerce
```

Comment:
Let's drop this. I didn't know we had this tag. I removed it from there repo now, so let's remove it from here.

Reply:
@osterman resolved